components/x11/mrxvt/Makefile

```diff
@@ -40,20 +40,18 @@ CFLAGS            +=	$(JPEG_CPPFLAGS) $(JPEG_CFLAGS)
 CXXFLAGS          +=	$(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)
 LDFLAGS           +=	$(JPEG_LDFLAGS)
 
+LDFLAGS			  +=	-lutempter
+
 CONFIGURE_OPTIONS +=	--sysconfdir=/etc
 CONFIGURE_OPTIONS +=	--enable-everything
```

Comment:
I see... the reason is that --enable-everything implies to try --enable-utempter.
All good :)